mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.tmpl

```diff
@@ -111,33 +111,35 @@ func TestAccComputeSubnetwork_update(t *testing.T) {
 			{
 				Config: testAccComputeSubnetwork_update1(cnName, "10.2.0.0/24", subnetworkName),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeSubnetworkExists(
-						t, "google_compute_subnetwork.network-with-private-google-access", &subnetwork),
+					testAccCheckComputeSubnetworkExists(t, "google_compute_subnetwork.network-with-private-google-access", &subnetwork),
 				),
 			},
```

Comment:
The config steps should all have ImportStateVerify steps after them.

```suggestion
			},
			{
				ResourceName:      "google_compute_subnetwork.network-with-private-google-access",
				ImportState:       true,
				ImportStateVerify: true,
			},
```

Reply:
as discussed, leaving this out for now because there's a bug in the resource where one of these steps is not updating in-place (and we have a separate test for the actual change at this point.)